src/main/java/com/danichef/rest/player/seats/PlayerSitImpl.java

```diff
@@ -1,15 +1,23 @@
 package com.danichef.rest.player.seats;
 
+import com.danichef.rest.Rest;
 import com.danichef.rest.utils.MessagesUtil;
 import org.bukkit.Location;
+import org.bukkit.NamespacedKey;
 import org.bukkit.entity.ArmorStand;
+import org.bukkit.entity.Entity;
 import org.bukkit.entity.Player;
+import org.bukkit.persistence.PersistentDataType;
 
 public class PlayerSitImpl implements PlayerSit {
 
+    private final NamespacedKey key;
+
+    public PlayerSitImpl(final Rest rest) { key = new NamespacedKey(rest, MessagesUtil.AS_KEY); }
```

Comment:
Providing the rest plugin instance here is not ideal usually.
Either provide just any [Plugin](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/plugin/Plugin.html) or ideally simply pass the [NamespacedKey ](https://hub.spigotmc.org/javadocs/spigot/org/bukkit/NamespacedKey.html) instance.